src/AlgebraicGeometry/Schemes/AffineVariety/Objects/Constructors.jl

```diff
@@ -32,7 +32,7 @@ julia> R, (x,y) = QQ[:x,:y]
 
 julia> variety(ideal([x,y]))
 Affine variety
-  in affine 2-space over QQ with coordinates x, y
+  in 𝔸² over QQ with coordinates x, y
```

Comment:
This looks like some `is_unicode_enabled` is not checked?

Reply:
Yes it is, I did not change the condition on the unicode checked